app.js

```diff
@@ -73,20 +100,33 @@ finishGameButton.addEventListener('click', async () => {
 // on load . . .
 window.addEventListener('', async () => {
     // display all past games (hint: call displayAllGames())
+    displayAllGames();
 });
 
 function displayCurrentGameEl() {
     // clear out the current game div
-    // change the label to show team one's name;
+    currentGameEl.textContent = '';
+    // change the label to show team one's name;s
+    teamOneLabel.textContent = currentGame.name1;
     // change the label to show team two's name;
+    teamTwoLabel.textContent = currentGame.name2;
     // call the render game function to create a game element
+    const renderedGame = renderGame(currentGame);
     // append the element to the cleared out current game div
+    currentGameEl.append(renderedGame);
+    
 }
 
-function displayAllGames() {
+async function displayAllGames() {
     // clear out the past games list in the DOM
+    pastGamesEl.textContent = '';
     // FETCH ALL GAMES from supabase
+    const games = await getGames();
     // loop through the past games
+    for (let game of games) {
+        const pastGame = renderGame(game);
+        pastGamesEl.append(pastGame);
+    }
     // render and append a past game for each past game in state
 }
 
```

Comment:
You should be able to just call displayAllGames() here to get your past games to display on load.